sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/AzureKeyVaultConfiguration.java

```diff
@@ -6,5 +6,5 @@
 class AzureKeyVaultConfiguration {
     //TODO: Eventually remove these hardcoded strings with https://github.com/Azure/azure-sdk-for-java/issues/3141
     static final String SDK_NAME = "Azure-Keyvault";
-    static final String SDK_VERSION = "4.0.0-preview.6";
+    static final String SDK_VERSION = "4.0.0-preview.6"; // {x-version-update;com.azure:azure-keyvault-secrets;current}
```

Comment:
this should be 4.0.0-preview.5

Reply:
@g2vinay Versions were updated yesterday by @sranthar and the azure-keyvault* artifacts were ticked up from 4.0.0-preview.5 to 4.0.0-preview.6.

Reply:
yeah, that was .a mistake, KV should be at 5

Reply:
What's showing highlighted above is outdated. The correct version, meaning the one you'd updated in version_client.txt is what's in these files now.